### ISSUE TITLE
Error objects in action payloads are eaten by toJSON before being displayed

### ISSUE THREAD

**Issue by jimmed** [martyjs/marty-devtools#8]
## Context

Let's say for example, I have an action creator like this:

```
doSomeMagic() {
  this.dispatch(Constants.MAGIC_STARTING);

  api.doMagicAsync()
    .then(result => this.dispatch(Constants.MAGIC_DONE, result)
    .catch(error => this.dispatch(Constants.MAGIC_FAILED, error);
}
```

Now let's imagine that my `api.doMagicAsync` returns a Promise which is rejected with an Error - this is caught by the `catch` block, and dispatches a `MAGIC_FAILED` action with the error as its payload.

If I want to debug this error, I can look at the action payload through the Marty dev tools. However, all I would see is an empty object.
## TL;DR

If I dispatch an action with an Error as a payload, Marty's dev tools displays the payload as `{}`.

I did a little digging through the source code, and essentially the problem is that the payload gets passed through the default serializer - in this case `JSON.parse(JSON.stringify(...))`; this just returns `{}`.

I should be able to fix this later today - stay tuned for a pull request unless someone else gets there first.
